collector/container/rhtap.Dockerfile

```diff
@@ -0,0 +1,153 @@
+ARG BUILD_DIR=/build
```

Comment:
This Dockerfile is very similar to `rhtap-slim.Dockerfile`:

```bash
$ diff collector/container/rhtap-slim.Dockerfile collector/container/rhtap.Dockerfile
100a101,104
> # Do NOT use follow_tag here, as we do not need or want collector to be rebuilt
> # with each drivers build (which may become very frequent)
> FROM brew.registry.redhat.io/rh-osbs/rhacs-drivers-build-rhel8:0.1.0 AS drivers-build
> # FROM registry-proxy.engineering.redhat.com/rh-osbs/rhacs-drivers-build-rhel8:0.1.0 AS drivers-build
132a137
> COPY --from=drivers-build /kernel-modules /kernel-modules
```

Reply:
Is it possible to parameterize the builds to reduce the dockerfile duplication?

Reply:
This is to be addressed in https://issues.redhat.com/browse/ROX-22394.